Tests/HummingbirdTests/URLEncodedForm/URLEncodedNodeTests.swift

```diff
@@ -104,22 +104,27 @@ class URLEncodedFormNodeTests: XCTestCase {
     }
 }
 
-extension URLEncodedFormNode: ExpressibleByStringLiteral {
+#if $RetroactiveAttribute
```

Comment:
```suggestion
#if hasFeature(RetroactiveAttribute)
```
See the other comment.